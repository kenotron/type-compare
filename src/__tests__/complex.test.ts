import { test } from "@jest/globals";
import { comapreHarness, getCasesByGroup } from "./compare-harness";

const group = "complex";

test.concurrent.each(getCasesByGroup(group))(`Test case "%s"`, async (testCase) => {
  comapreHarness(group, testCase);
});
