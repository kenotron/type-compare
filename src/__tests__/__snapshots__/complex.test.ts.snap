// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test case "declared-non-exported-type-change" 1`] = `
{
  "minChangeType": "major",
}
`;
